lib/matplotlib/axes/_axes.py

```diff
@@ -4795,7 +4795,7 @@ def arrow(self, x, y, dx, dy, **kwargs):
         Parameters
         ----------
         x, y : float
-            The x/y-coordinate of the arrow base.
+            The x and y coordinates of the arrow base.
         dx, dy : float
             The length of the arrow along x/y-direction.
```

Comment:
If we remove `x/y` notation, please change this as well.

Reply:
Done. Would you recommend changing it in other places like [here](https://github.com/DumbMachine/matplotlib/blob/patch-1/lib/matplotlib/axes/_axes.py#L1660)